src/can-observable-array.js

```diff
@@ -10,18 +10,27 @@ const {
 } = require("./helpers");
 const ProxyArray = require("./proxy-array")();
 const queues = require("can-queues");
+const { normalizeTypeDefinition } = require("can-observable-mixin/dist/define-helpers");
```

Comment:
this has to specificy `/dist/` ?

Reply:
Yeah it won't load without it. Is there an alternative?